index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dany's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+                <a href="#"class="Logo">My Logo</a>
+                <nav class="Navbar">
+                </nav>
+                <div class="menu"><a href="#"><img src="img/humburger-btn.svg" alt="menu"></a></div>
```

Comment:
- [ ] KIndly make sure to wrap the hamburger icon into the navigation tag `nav`. At this moment inside the `nav` tags there is nothing.